lisa/operating_system.py

```diff
@@ -765,14 +776,6 @@ class Debian(Linux):
         r"\s+(?P<metadata>.*)\s*"
     )
 
-    """ Package: dpdk
```

Comment:
Why remove this logic?

Reply:
With the new dpkg query being used, we can get the version string directly, so this regex is no longer needed to extract the version string